RecoJets/JetProducers/interface/NjettinessAdder.h

```diff
@@ -44,6 +45,7 @@ class NjettinessAdder : public edm::stream::EDProducer<> {
   ~NjettinessAdder() override {}
 
   void produce(edm::Event& iEvent, const edm::EventSetup& iSetup) override;
+  void addFJParticle(std::vector<fastjet::PseudoJet>& FJparticles, const reco::CandidatePtr& dp) const;
```

Comment:
FJparticles
start local variables and data members with lower case letter. Capitalization is reserved for class names.